app/javascript/components/widgets/numbers/fallingblocks.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+/* global UnityLoader */
+
+function shuffleArray(oldArray) {
+  const array = oldArray;
+  for (let i = array.length - 1; i > 0; i -= 1) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+}
+
+class FallingBlocks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.blocks = [];
+    for (let x = 0; x < props.pr; x += 1) {
+      this.blocks.push('pr');
+    }
+    for (let x = 0; x < props.commit; x += 1) {
+      this.blocks.push('commit');
+    }
+    for (let x = 0; x < props.slack; x += 1) {
+      this.blocks.push('slack');
+    }
+
+    shuffleArray(this.blocks);
+  }
+  componentDidMount() {
+    this.fallingBlocks = UnityLoader.instantiate(
+      'falling-blocks-canvas',
+      'unity/webgl.json',
+      {
+        Module: {
+          // wait for unity splash screen to go away before
+          // adding blocks
+          preRun: [() => setTimeout(this.addBlocks, 4500)],
```

Comment:
It looks like Unity provides a flag to indicate if the splash screen is finished: https://docs.unity3d.com/ScriptReference/Rendering.SplashScreen-isFinished.html

Reply:
```suggestion
          preRun: [() => setTimeout(this.addBlocks, 1500)],
```